COMMITTERS.rst

```diff
@@ -188,3 +188,4 @@ To be able to merge PRs, committers have to integrate their GitHub ID with Apach
 4.  Wait at least 30  minutes for an email inviting you to Apache GitHub Organization and accept invitation.
 5.  After accepting the GitHub Invitation verify that you are a member of the `Airflow committers team on GitHub <https://github.com/orgs/apache/teams/airflow-committers>`__.
 6.  Ask in ``#internal-airflow-ci-cd`` channel to be `configured in self-hosted runners <https://github.com/apache/airflow-ci-infra/blob/main/scripts/list_committers>`_ by the CI maintainers
+7.  Open a PR to include your Github alias in `ci.yml <https://github.com/apache/airflow/blob/main/.github/workflows/ci.yml#L72>`__ as well as your name and alias in `project.rst <https://github.com/apache/airflow/blob/main/docs/apache-airflow/project.rst>`__.
```

Comment:
```suggestion
7.  After confirmation step 6 is done, open a PR to include your Github alias in `ci.yml <https://github.com/apache/airflow/blob/main/.github/workflows/ci.yml#L72>`__ as well as your name and alias in `project.rst <https://github.com/apache/airflow/blob/main/docs/apache-airflow/project.rst>`__.
```
If not mistaken its important that 7 will happen after otherwise your PRs will not get CI resources

Reply:
Sure, I was working with the assumption that the individual is working through the steps in order, but it's better to be safe than sorry :+1: